packages/common/src/utils/wallet.ts

```diff
@@ -117,6 +117,7 @@ export const convertJSBIToAmountObject = (amount: JSBI, decimals: number) => {
     : quotient.toString()
   return {
     amount: JSBI.toNumber(amount),
+    amountString: JSBI.toString(),
```

Comment:
should this include the amount? looks like it does not

Reply:
yes! this is awkward...